src/state/codeSlice.ts

```diff
@@ -0,0 +1,23 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
```

Comment:
Additionally, any time you make a new slice, you need to add it in store.ts
Otherwise it will not actually be a part of the state